include/mapbox/cheap_ruler.hpp

```diff
@@ -193,6 +201,11 @@ class CheapRuler {
         double minDist = std::numeric_limits<double>::infinity();
         double minX = 0., minY = 0., minI = 0., minT = 0.;
 
+        if (line.empty()) {
+            return std::make_tuple(
+                point(minX, minY), minI, ::fmax(0., ::fmin(1., minT)));
```

Comment:
Nit: this is slightly redundant — `std::make_tuple(point(0., 0.), 0., 0.)` would be simpler / more explicit